posts/2024-xx-xx-spring-boot-3.adoc

```diff
@@ -0,0 +1,223 @@
+---
+layout: post
+title: "Running a Spring Boot 3.x application WAR file on Liberty"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/scottkurz
+author_github: https://github.com/scottkurz
+seo-title: Running a Spring Boot 3.x application WAR file on Liberty
+seo-description: TODO In version 23.0.0.9 and later, you can easily run a Spring Boot 3.x application from a WAR file on Open Liberty. This post use the Spring Boot Petclinic sample application and the Liberty Maven plugin to create and run a WAR-based Spring Boot 3.1 application on Open Liberty. 
+blog_description: TODO In version 23.0.0.9 and later, you can easily run a Spring Boot 3.x application from a WAR file on Open Liberty. This post uses the Spring Boot Petclinic sample application and the Liberty Maven plugin to create and run a WAR-based Spring Boot 3.1 application on Open Liberty. 
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Running a Spring Boot 3.x application WAR file on Liberty
+Scott Kurz <https://github.com/scottkurz>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+As of Liberty version 23.0.0.9 there are now two ways to deploy Spring Boot WAR files on Liberty.
```

Comment:
I know this hasn't been addressed in any of our other blog posts/guide about SB support yet either, but I think this sentence should be followed by answering the question of "why would I do that [deploy a Spring Boot WAR on Liberty]?". Without it, yes, the post is providing some helpful documentation for people who need it, but a blog post should provide more than that and it would be good to also convince readers that it's a valuable thing to do for some use cases.

Reply:
eg to benefit from InstantOn for a faster startup whilst maintaining throughput and not having to modify the app?

Reply:
The answer to "why" might depend on your baseline alternative

 * not going to get into why choose WAR but assume they know.  I even feel like "..to take advantage of the Spring and Spring Boot libraries " is too obvious to mention but leaving it for now as a compromise
 * follow-up.. this good to reference: 

That answers why SB on Liberty.

As far as why "optimized deployment"?  That's addressed with:

>  This deployment adds some additional configuration options, and provides advantages for efficiently building Docker container images

OTOH, InstantOn is supposed to work with both deployments.